Tools/build/check_warnings.py

```diff
@@ -114,24 +114,28 @@ def get_unexpected_improvements(
 def main(argv: list[str] | None = None) -> int:
     parser = argparse.ArgumentParser()
     parser.add_argument(
+        "-c",
         "--compiler-output-file-path",
         type=str,
         required=True,
         help="Path to the compiler output file",
     )
     parser.add_argument(
+        "-i",
         "--warning-ignore-file-path",
         type=str,
         required=True,
```

Comment:
This is actually something I wonder, but is this argument actually required? (at least, for me as a local user, I don't want to pass a dummy one, but I can live with that if you prefer to force it).

Reply:
Sure this could be optional, and in the absence of a warning ignore file the script can just assume there are no files to ignore. I will add that.

Reply:
Left a review on that PR to remind me to add that ability: https://github.com/python/cpython/pull/122211